it/uk/gov/hmrc/claimvatenrolmentfrontend/views/CaptureBusinessPostcodeViewTests.scala

```diff
@@ -59,5 +63,17 @@ trait CaptureBusinessPostcodeViewTests {
       doc.getSubmitButton.first.text mustBe Base.continue
     }
   }
+  def testCaptureVatRegistrationPostcodeRedirectViewTests(result: => WSResponse,
```

Comment:
Don't need the redirect in the name here. Maybe call it testCaptureBusinessPostcodeMissingErrorViewTests